### PR TITLE
refactor(Colors): migrate colors and use UIColor extension

### PR DESCRIPTION
**Pull request by achtenhagen**
Highlights in this PR:
- Adds new `UIColor` extension for simpler color management.
- Migrates remaining colors from `Blockchain-Prefix.pch` file.

### CODE REVIEW

File: Blockchain/ModalPresenter.swift [hunk 1]
```diff
@@ -150,11 +150,7 @@ typealias OnModalResumed = () -> Void
         //: due to bottom insets on the iPhone X
         // TODO: remove this stopgap solution when the modal view presenter is deprecated
         if content is PairingInstructionsView {
-            if #available(iOS 11.0, *) {
-                modalViewToShow.backgroundColor = UIColor(named: "ColorGray1")
-            } else {
-                modalViewToShow.backgroundColor = Constants.Colors.ColorGray1
-            }
+            modalViewToShow.backgroundColor = UIColor.gray1
```
Comment (jstxx) [inline Blockchain/ModalPresenter.swift]: Don't need the UIColor prefix here

File: Blockchain/PrivateKeyReader/PrivateKeyReader.swift [hunk 2]
```diff
@@ -120,11 +120,7 @@ final class PrivateKeyReader: UIViewController & AVCaptureMetadataOutputObjectsD
         self.view.frame = viewFrame
 
         let topBar = UIView(frame: CGRect(x: 0, y: 0, width: self.view.frame.size.width, height: Constants.Measurements.DefaultHeaderHeight))
-        if #available(iOS 11.0, *) {
-            topBar.backgroundColor = UIColor(named: "ColorBrandPrimary")
-        } else {
-            topBar.backgroundColor = Constants.Colors.ColorBrandPrimary
-        }
+        topBar.backgroundColor = UIColor.brandPrimary
```
Comment (jstxx) [inline Blockchain/PrivateKeyReader/PrivateKeyReader.swift]: same here

File: Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift [hunk 3]
```diff
@@ -48,19 +48,19 @@ import UIKit
     override func awakeFromNib() {
         super.awakeFromNib()
 
-        imageViewAsset.tintColor = Constants.Colors.ColorGray5
+        imageViewAsset.tintColor = UIColor.gray5
```
Comment (jstxx) [inline Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift]: Same as earlier

File: Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift [hunk 4]
```diff
@@ -48,19 +48,19 @@ import UIKit
...
             name: Constants.FontNames.montserratLight,
             size: Constants.FontSizes.Small
         )
         buttonRequest.layer.borderWidth = 1.0
-        buttonRequest.layer.borderColor = Constants.Colors.ColorBrandSecondary.cgColor
+        buttonRequest.layer.borderColor = UIColor.brandSecondary.cgColor
         buttonRequest.layer.cornerRadius = 8.0
-        buttonRequest.backgroundColor = Constants.Colors.ColorBrandQuaternary
-        buttonRequest.setTitleColor(Constants.Colors.ColorBrandSecondary, for: .normal)
+        buttonRequest.backgroundColor = UIColor.brandQuaternary
```
Comment (jstxx) [inline Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift]: and here

File: Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift [hunk 5]
```diff
@@ -48,19 +48,19 @@ import UIKit
...
+        buttonRequest.setTitleColor(UIColor.brandSecondary, for: .normal)
 
-        labelAddress.textColor = Constants.Colors.ColorGray5
+        labelAddress.textColor = UIColor.gray5
```
Comment (jstxx) [inline Blockchain/Swipe to Receive/Views/SwipeToReceiveAddressView.swift]: also here

**Comment by achtenhagen** [comment]
Thanks for reviewing @jstxx, I will make the suggested changes to this PR.

**Pull request merged**